server/routes/index.js

```diff
@@ -1,8 +1,8 @@
 import express from 'express';
-import user from './user';
+import api from './api/v1';
```

Comment:
All the routes should be in the routes folder. You shouldn't create the '/api/v1' folder. Where '/api/v1' comes in is in the prefix for the routes.

Reply:
Treated accordingly